saleor/payment/gateway.py

```diff
@@ -76,12 +74,10 @@ def wrapped(payment: Payment, *args, **kwargs):
 def process_payment(
     payment: Payment,
     token: str,
+    plugin_manager: "PluginsManager",
```

Comment:
I think in most places in the codebase this argument is called `manager`. I would rename it to `manager` here and in other functions that use a different name.